### PR TITLE
Use latest platform SDK

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
Integrated platform SDK `0.32.0-alpha.1`

### CODE REVIEW

**Review by cody-littley — APPROVED** [review]

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/services/stream/RecordStreamManager.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
  */
 package com.hedera.services.stream;
 
+import static com.swirlds.common.threading.manager.AdHocThreadManager.getStaticThreadManager;
```
Comment (mhess-swl) [inline hedera-node/hedera-mono-service/src/main/java/com/hedera/services/stream/RecordStreamManager.java:18]: `AdHocThreadManager` sounds like a fun thing to put into prod 😂

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/migration/AccountStorageAdapterTest.java [hunk 2]
```diff
@@ -214,7 +216,10 @@ void onDiskPropagatesInterruption() throws InterruptedException {
         willThrow(InterruptedException.class)
                 .given(virtualMapDataAccess)
                 .extractVirtualMapData(
-                        eq(onDiskAccounts), any(InterruptableConsumer.class), eq(32));
+                        getStaticThreadManager(),
```
Comment (mhess-swl) [inline hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/migration/AccountStorageAdapterTest.java:219]: I'm surprised this works. Doesn't mockito usually want an `eq()` around `getStaticThreadManager()` if there are `eq()` or `any()` on other params?

File: settings.gradle.kts [hunk 3]
```diff
@@ -68,7 +68,7 @@ dependencyResolutionManagement {
             version("log4j-version", "2.17.2")
             version("netty-version", "4.1.66.Final")
             version("protobuf-java-version", "3.19.4")
-            version("swirlds-version", "0.31.0-alpha.1")
+            version("swirlds-version", "0.32.0-alpha.0")
```
Comment (mhess-swl) [inline settings.gradle.kts:71]: I was told it's `0.32.0-alpha.1` (with a 1 at the end)?

File: hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/migration/AccountStorageAdapterTest.java (hunk 2 above)
Comment (Neeharika-Sompalli) [reply hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/migration/AccountStorageAdapterTest.java]: It did not work. 🙂  There were few other test failures. Fixed some of them

File: settings.gradle.kts (hunk 3 above)
Comment (Neeharika-Sompalli) [reply settings.gradle.kts]: Ah yes. Thanks ! Fixed it.

**Comment by Neeharika-Sompalli** [comment]
Replaced with this PR https://github.com/hashgraph/hedera-services/pull/4117

**Pull request closed without merging**